image_acquisition:
  input_type: "video" #camera_usb, video
  
  camera_usb: 
      camera_node: 1
  
  video:
      video_node: 1
      video_path_0: "/home/sirhawk/catkin_ws/src/image_pipeline/test_data/video/test_1.mp4"
  
  frequency: 10